lib/spack/spack/environment/environment.py

```diff
@@ -177,10 +186,7 @@ def valid_env_name(name):
 def validate_env_name(name):
     if not valid_env_name(name):
         raise ValueError(
-            (
-                "'%s': names must start with a letter, and only contain "
-                "letters, numbers, _, and -."
-            )
+            ("'%s': names must start with a letter, and only contain letters, numbers, _, and -.")
```

Comment:
If we have to change this for style, can we just change it to an fstring while we're at it?

Reply:
Fixed.